setup.sh

```diff
@@ -40,7 +40,7 @@ function installTools ()
 
 function installThirdParty ()
 {
-  runAndAssertCmd "$NPM_BIN_DIR/bower install --allow-root --config.interactive=false"
+  runAndAssertCmd "$NPM_BIN_DIR/bower install --allow-root --config.interactive=false --config.directory=build/third_party/bower"
```

Comment:
Thanks :-)
FYI, I also have this as a separate pull request: https://github.com/uProxy/uproxy/pull/2477.
I can merge that separately if you prefer.